be/src/common/config.h

```diff
@@ -597,6 +597,8 @@ CONF_mInt32(zone_map_row_num_threshold, "20");
 //    Trace = 6
 CONF_Int32(aws_log_level, "3");
 
+// default level of MemTracker to show in webpage 
```

Comment:
Explain whether this value is higher and more detailed or lower and more detailed.